spec.bs

```diff
@@ -559,6 +611,8 @@ The [=steps to respond to a payment request=] for this payment method, for a giv
     → |publicKeyOpts|]».
 
     Note: This triggers [[webauthn-3]]'s [[webauthn-3#sctn-getAssertion|Get]] behavior
+    
+    Q: Which version of WebAuthn will SPC require? Can we make it work for Level 1, or is Level 2 or 3 required?
```

Comment:
I think L2 might be required for discoverable credentials?

Reply:
Will you be able tp make it work without that or is discoverable credentials a minimum requirement?

Reply:
I would consider them a minimum requirement, in order to move from a browser-based cache to a platform-based cache and thus avoid SPC credentials being per-browser on a given device. See https://github.com/w3c/secure-payment-confirmation/issues/92